README.md

```diff
@@ -146,31 +191,8 @@ To control keys and certificates used for TLS, emmy CLI programs use several fla
 * `--cert` which expects the path to server's certificate in PEM format, 
 * `--key` which expects the path to server's private key file.
 
-The same flags can be used with `emmy example` program. If they are omitted, files in `test/testdata` directory are used.
-
 On the other hand, we can provide `emmy client` with the `--caCert` flag, which expects the certificate of the CA that issued emmy server's certificate (in PEM format). Again, if this flag is omitted, the certificate in `test/testdata` directory is used.
```

Comment:
`--caCert` -> `--cacert`

Reply:
Sorry, these flags, again... :smile: You have a good eye :+1: